src/main.c

```diff
@@ -60,7 +61,8 @@ int main() {
     // get current states
     temperature = api.getTemperature();
 
-    _log("%f", temperature);
+    printf("%f\n", temperature);
```

Comment:
what's the rationale behind using `printf` instead of `_log`?

Reply:
Wir müssen uns da mit den Elektrotechnikern enger absprechen, was die für ein Ausgabeformat haben wollen. Hatte das glaub rausgenommen, weil es für Kotaro leichter war, nur die Temperaturwerte zu nehmen und nicht mit der Zeit. Da sollten wir vllt wirklich mal ein Skript zu erstellen. Als modus operandi würde ich vorschlagen, das wir ignorieren, was da konkret drin steht. Bei den aktuellen Experimenten, die die Elektrotechniker fahren müssen, wird sich das sowieso immer wieder ändern.

Reply:
As a default mode, I don't see any advantages to using `printf`. We are using `_log` specifically to include additional information in an unified way across the codebase.  
w.r.t. to different modes, I'll work on implementing some menu system to run different programs